/dev/kmsg inaccessible during remote bazel execution

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/76924/pull-kubernetes-bazel-test-rbe/1121234523384713218

```
FAIL: //pkg/kubelet/oom:go_default_test (see /bazel-scratch/.cache/bazel/_bazel_root/7989b31489f31aee54f32688da2f0120/execroot/__main__/bazel-out/k8-fastbuild/testlogs/pkg/kubelet/oom/go_default_test/test.log)
INFO: From Testing //pkg/kubelet/oom:go_default_test:
==================== Test output for //pkg/kubelet/oom:go_default_test:
--- FAIL: TestBasic (0.00s)
oom_watcher_linux_test.go:32:
Error Trace:	oom_watcher_linux_test.go:32
Error: Received unexpected error:
open /dev/kmsg: no such file or directory
Test: TestBasic
FAIL
```

https://github.com/kubernetes/kubernetes/blob/a0d82783465019d36be3429552e6613270f9d1e9/vendor/github.com/euank/go-kmsg-parser/kmsgparser/kmsgparser.go#L59-L60

https://github.com/kubernetes/kubernetes/blob/a0d82783465019d36be3429552e6613270f9d1e9/vendor/github.com/google/cadvisor/utils/oomparser/oomparser.go#L130-L133

https://github.com/kubernetes/kubernetes/blob/a0d82783465019d36be3429552e6613270f9d1e9/pkg/kubelet/oom/oom_watcher_linux.go#L47-L48

https://github.com/kubernetes/kubernetes/blob/a0d82783465019d36be3429552e6613270f9d1e9/pkg/kubelet/oom/oom_watcher_linux_test.go#L31-L32

@kubernetes/sig-testing 
/assign

--- a/build/root/BUILD.root
+++ b/build/root/BUILD.root
@@ -110,10 +110,15 @@ genrule(
 platform(
     name = "rbe_with_network",
     parents = ["@rbe_default//config:platform"],
+    # https://cloud.google.com/remote-build-execution/docs/remote-execution-environment#remote_execution_properties
     remote_execution_properties = """
       properties: {
         name: "dockerNetwork"
         value: "standard"
+      }
+      properties: {
+        name: "dockerPrivileged"
+        value: "true"
       }
       {PARENT_REMOTE_EXECUTION_PROPERTIES}
     """,

